Dockerfile.txt

```diff
@@ -0,0 +1,67 @@
+FROM debian:bookworm
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+
+RUN apt-get -y dist-upgrade
+
+RUN apt-get -y --no-install-recommends install \
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl \
+    doxygen \
+    g++ \
+    gdb \
+    git \
+    gnupg \
+    gpg \
+    less \
+    libabsl-dev \
+    libboost-all-dev \
+    libgrpc++-dev \
+    libprotobuf-dev \
+    libssl-dev \
+    libxtensor-dev \
+    libusb-1.0-0-dev \
+    libudev-dev \
+    libgtk-3-dev \ 
+    ninja-build \
+    pkg-config \
+    protobuf-compiler-grpc \
+    software-properties-common \
+    sudo \
+    wget
+
+RUN mkdir -p /root/opt/src
+
+# install appimage-builder
+RUN apt install -y \
+    binutils \
+    coreutils \
+    desktop-file-utils \
+    fakeroot \
+    fuse \
+    libgdk-pixbuf2.0-dev \
+    patchelf \
+    python3-pip python3-setuptools \
+    squashfs-tools \
+    strace \
+    util-linux zsync
+
+RUN apt install -y python3.11-venv
+
+RUN pip3 install -U pip setuptools urllib3==1.26.12 requests==2.26.0 --break-system-packages
+
+# install appimage-builder
+RUN pip3 install --break-system-packages git+https://github.com/AppImageCrafters/appimage-builder.git@f38699ef3644fa5409a5a262b7b6d99d6fb85db9
```

Comment:
Ideally everything up to here could be part of some kind of viam standard developer container